stages/org.osbuild.update-crypto-policies

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/python3
+import os
 import subprocess
 import sys
 
 from osbuild import api
+from osbuild.util.mnt import MountGuard, MountPermissions
 
 
 def main(tree, options):
     policy = options["policy"]
 
-    cmd = ["/usr/sbin/chroot", tree,
-           "/usr/bin/update-crypto-policies", "--set", policy]
+    with MountGuard() as mg:
+        source = "/proc/self/mountinfo"
```

Comment:
Maybe worth adding a comment here? (and out of curiosity) what exactly is the mountpoint that FIPS provides and update-crypto-policies is looking for?

Reply:
You're right, I'll add a comment with some details from the commit message.